values.yaml

```diff
@@ -40,9 +40,17 @@ ingress:
   # be left blank, and the user can still access the platform from the external
   # IP or a DNS name that resolves to the external IP address.
   host: ""
-  # ingress.loadBalancerIP sets the external IP address of the ingress to the
+  # ingress.loadBalancerIP -- sets the external IP address of the ingress to the
   # provided value.
   loadBalancerIP: ""
+  # ingress.loadBalancerSourceRanges -- If specified and supported by the
+  # platform, the traffic through the cloud-provider load-balancer will be
+  # restricted to the specified client IPs. This field will be ignored if the
+  # cloud-provider does not support the feature.
+  #
+  # More info:
+  # https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/
```

Comment:
seems to be a dead link?

Reply:
bold of me to assume the kubernetes docs contain links that work :rofl: